internal/domain/users/httphandler/user_handler_test.go

```diff
@@ -108,23 +108,23 @@ func (s *UserHandlerTestSuite) TestSaveUser_WhenSomethingFails() {
 	}{
 		{
 			name:           "should append ValidationError to the context when validator fails",
-			validatorError: errs.NewValidationError([]string{errs.ErrorEmailFormat, errs.ErrorAgeMinimum}),
+			validatorError: onboardingerrors.NewValidationError([]string{onboardingerrors.ErrorEmailFormat, onboardingerrors.ErrorAgeMinimum}),
 			repoError:      nil,
-			expected:       errs.NewValidationError([]string{errs.ErrorEmailFormat, errs.ErrorAgeMinimum}),
+			expected:       onboardingerrors.NewValidationError([]string{onboardingerrors.ErrorEmailFormat, onboardingerrors.ErrorAgeMinimum}),
 		},
 		{
 			name:           "should append UnexpectedError to the context when repo fails",
 			validatorError: nil,
-			repoError:      errs.NewUnexpectedError(errors.New("something is wrong")),
-			expected:       errs.NewUnexpectedError(errors.New("something is wrong")),
+			repoError:      onboardingerrors.NewUnexpectedError(errors.New("something is wrong")),
+			expected:       onboardingerrors.NewUnexpectedError(errors.New("something is wrong")),
 		},
 	}
 
 	for _, tc := range testCases {
 		s.Run(tc.name, func() {
 			response := httptest.NewRecorder()
 			ctx, _ := gin.CreateTestContext(response)
-			user := models.NewUser("1", "John", "Doe", "john.doe@example.com", 18)
+			user := users.NewUser("1", "John", "Doe", "john.doe@example.com", 18)
```

Comment:
I would just simplify this function name to users.New() what is being created is implicit in the package name 👍

Reply:
Thanks, addressed this in the new commit